Assets/Scripts/WorldEngine/Terrain/TerrainCell.cs

```diff
@@ -91,7 +91,8 @@ public class TerrainCell
     public float Buffer3 = 0;
     public int RiverId = -1;
     public float RiverLength = 0;
-    public bool DrainageDone = false;
```

Comment:
Why did this change to true?

Reply:
Because for most cells there's no need to do drainage calculations if it receives no rainfall. If a cell receives rainfall or drainage from other cells then the value will be changed to false as needed.